### PR TITLE
Basic Theme

### PR DESCRIPTION
**Pull request by revett**
### Problem

No real content or styling had been added to the page as of yet.

### Solution

- Add sidebar.
- Hard-code 3 videos.
- On hover effects.
- Click effects.

image

### Notes

- **Remember** to tag your merge commit after merging 👍

### CODE REVIEW

**Comment by revett** [comment]
[[SSA 0.5.0] Basic Theme](https://trello.com/c/PQ9rvmpS/1160-ssa-0-5-0-basic-theme)

**Comment by revett** [comment]
@stevenjack ready for review in the AM

File: src/components/presentational/sidebar_header.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+export default class SidebarHeader extends React.Component {
+  constructor() {
+    super();
+    this.state = {};
+  }
+  render() {
+    return (
+      <div className="SidebarHeader">
+        <a href="https://clients.vidsy.co">Vidsy</a>
```
Comment (stevenjack) [inline src/components/presentational/sidebar_header.js]: Add config as prop not hardcoded

Comment (revett) [inline src/components/presentational/sidebar_header.js]: I have added a number of cards to Trello, to cover this in another PR.

**Pull request merged**